RawSpeed/DngDecoder.cpp

```diff
@@ -230,17 +234,23 @@ RawImage DngDecoder::decodeRawInternal() {
       } catch (TiffParserException) {
         ThrowRDE("DNG Decoder: Unsupported format, uncompressed with no strips.");
       }
-    } else if (compression == 7 || compression == 0x884c) {
+    } else if (compression == 7 || compression == 8 || compression == 0x884c) {
```

Comment:
In my opinion that's meaningless, since it decoding will fail later, and all extra ifdefs make code readability worse, but OK.